From 62be0b30d407d621a5b195cec9b0a4fa91944c0c Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 16:07:34 +0100
Subject: [PORT FROM MAIN][BT] Multivendor support: adding libbt-vendor conf for x86 targets

BZ: 76413 72228 99491 103267

Including the specific Intel configuration file for all x86 targets
in libbt-vendor configuration.

Some compile-time configuration variables for Bluetooth are board
specific.

As a consequence, the Bluedroid configuration files are created in
product device specific configuration folders.

Change-Id: Ia4b1a32235e8d219bea4c7b79fc34f6ace7cfb9a
Orig-Change-Id: I788675811f96b45e28b00c18cf597ad3025fd195
Orig-Change-Id: I6ee8fca5ec7033388e79ff447418386344a8453a
Orig-Change-Id: Ic18ade5b3f480d8c99911d77054f14bdf827a1b6
Orig-Change-Id: I65d36b935f8cb087389a9575c108b891a103e4fb
Feature: BT
Category: Integration
Purpose: ABSP-Adapter
Origin: Internal
Upstream-candidate: no
Signed-off-by: Raffaele Aquilone <raquiloX>
Signed-off-by: Julien Gros <julien.gros@intel.com>
Reviewed-on: http://android.intel.com:8080/85372
Reviewed-on: http://android.intel.com:8080/85605
Reviewed-on: http://android.intel.com:8080/101596
Reviewed-on: http://android.intel.com:8080/104375
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Biselli, MirkoX <mirkox.biselli@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Isaksen, FrodeX <frodex.isaksen@intel.com>
Reviewed-by: Combes, Patrick <patrick.combes@intel.com>
Reviewed-by: Vamour, Cedric <cedric.vamour@intel.com>
Reviewed-by: Fert, Laurent <laurent.fert@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Imberton, Guilhem <guilhem.imberton@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 libbt/conf/intel/baylake/bt_vendor.conf     |    5 +++++
 libbt/conf/intel/bodegabay/bt_vendor.conf   |    5 +++++
 libbt/conf/intel/ctpscaleht/bt_vendor.conf  |    5 +++++
 libbt/conf/intel/ctpscalelt/bt_vendor.conf  |    5 +++++
 libbt/conf/intel/redhookbay/bt_vendor.conf  |    5 +++++
 libbt/conf/intel/saltbay_pr0/bt_vendor.conf |    5 +++++
 libbt/conf/intel/saltbay_pr1/bt_vendor.conf |    5 +++++
 libbt/conf/intel/victoriabay/bt_vendor.conf |    5 +++++
 8 files changed, 40 insertions(+), 0 deletions(-)
 create mode 100644 libbt/conf/intel/baylake/bt_vendor.conf
 create mode 100644 libbt/conf/intel/bodegabay/bt_vendor.conf
 create mode 100644 libbt/conf/intel/ctpscaleht/bt_vendor.conf
 create mode 100644 libbt/conf/intel/ctpscalelt/bt_vendor.conf
 create mode 100644 libbt/conf/intel/redhookbay/bt_vendor.conf
 create mode 100644 libbt/conf/intel/saltbay_pr0/bt_vendor.conf
 create mode 100644 libbt/conf/intel/saltbay_pr1/bt_vendor.conf
 create mode 100644 libbt/conf/intel/victoriabay/bt_vendor.conf

diff --git a/libbt/conf/intel/baylake/bt_vendor.conf b/libbt/conf/intel/baylake/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/baylake/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/bodegabay/bt_vendor.conf b/libbt/conf/intel/bodegabay/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/bodegabay/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/ctpscaleht/bt_vendor.conf b/libbt/conf/intel/ctpscaleht/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/ctpscaleht/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/ctpscalelt/bt_vendor.conf b/libbt/conf/intel/ctpscalelt/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/ctpscalelt/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/redhookbay/bt_vendor.conf b/libbt/conf/intel/redhookbay/bt_vendor.conf
new file mode 100644
index 0000000..26a7e5f
--- /dev/null
+++ b/libbt/conf/intel/redhookbay/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/hci_tty
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/saltbay_pr0/bt_vendor.conf b/libbt/conf/intel/saltbay_pr0/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/saltbay_pr0/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/saltbay_pr1/bt_vendor.conf b/libbt/conf/intel/saltbay_pr1/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/saltbay_pr1/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/victoriabay/bt_vendor.conf b/libbt/conf/intel/victoriabay/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/victoriabay/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
-- 
1.7.4.1

